app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -755,12 +755,18 @@ def rbac_field_changed(rec_type)
     end
 
     @edit[:new][:group] = rbac_user_get_group_ids.map(&:to_i) if rec_type == "user"
-    session[:changed] = changed = (@edit[:new] != @edit[:current])
-    bad = false
-    if rec_type == "group"
-      bad = (@edit[:new][:role].blank? || @edit[:new][:group_tenant].blank?)
-    end
-    bad = @edit[:new][:name].blank? if rec_type == 'role'
+
+    bad = case rec_type
+          when 'group'
+            @edit[:new][:role].blank? || @edit[:new][:group_tenant].blank? || @edit[:new][:description].blank?
```

Comment:
I think this is prettier, but I leave it up to you :wink: 
```suggestion
            @edit[:new].values_at(:role, :group_tenant, :description).any?(&:blank?)
```